include/la/avdecc/utils.hpp

```diff
@@ -368,7 +362,7 @@ class EnumBitfield final
 	}
 
 	/** Clears the specified flag. If passed value is not valid (not exactly one bit set), this leads to undefined behavior. */
-	EnumBitfield& reset(value_type const flag) noexcept
+	constexpr EnumBitfield& reset(value_type const flag) noexcept
```

Comment:
This used to cause compilation issues on some VisualStudio version, but it's now OK with the version I'm using, so I'll add this, not that it will change much :)

Reply:
This is actually called from another function that is in turn marked `constexpr`. This not being `constexpr` itself caused a compilation failure for me.

Reply:
VS has a long history of not being compatible with the c++ standard, especially with the constexpr keyword (and highly depending on the rapid releases of VS).
This is not just a matter of testing one compiler, I need to make sure it works on many of them flawlessly, the code being use in production code.

Reply:
No argument here. What I'm saying is: This fixes an actual bug for me, calling a non-constexpr function in a constexpr context. Precisely to make this compatible with a compiler. It is not just making a function constexpr "cause we can". If this doesn't work in VS the solution will have to be removing constexpr elsewhere.

Reply:
Well, in my case it was merely the unit tests that did not compile (the same that are in the repo), so it was just gtest function and I don't see them as being constexpr :)

Anyway, since it seems fixed now, everything is good (using VS 15.9 and 16.0 preview), this will be pushed in an upcoming commit!